app/controllers/katello/api/v2/errata_controller.rb

```diff
@@ -21,8 +21,10 @@ def available_for_content_view_filter(filter, collection)
       collection = filter_by_content_view(filter, collection)
       ids = Katello::ContentViewErratumFilterRule.where(:content_view_filter_id => filter.id).pluck("errata_id")
       collection = collection.where("errata_id not in (?)", ids) unless ids.empty?
-      collection = collection.where('issued  >= ?', params[:start_date]) if params[:start_date]
-      collection = collection.where('issued  <= ?', params[:end_date]) if params[:end_date]
+      date_type = params[:date_type].present? ? params[:date_type] : ContentViewErratumFilterRule::ISSUED
```

Comment:
Maybe put this before the first collection statements so they can be parsed visually all together?